src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -184,7 +184,7 @@ struct SnapshotTestSetup : TestChain100Setup {
         {
             LOCK(::cs_main);
             BOOST_CHECK(!chainman.IsSnapshotValidated());
-            BOOST_CHECK(!node::FindSnapshotChainstateDir());
+            BOOST_CHECK(!node::FindSnapshotChainstateDir(m_args.GetDataDirNet()));
```

Comment:
In commit "refactor: Add path argument to FindSnapshotChainstateDir" (8789b11114b4bd6c7ee727dffbc75a6bdf20dd27)

Throughout this file, it would make tests clearer and reliant on details of test setup if the new
`m_args.GetDataDirNet()` calls were replaced with `chainman.m_options.datadir`

Reply:
@TheCharlatan did you want to open a followup for this?

Reply:
Yes.